bundle-kubeflow.yaml

```diff
@@ -0,0 +1,59 @@
+bundle: kubernetes
+name: kubeflow
+applications:
+  admission-webhook:             { charm: admission-webhook,       channel: latest/edge, scale: 1 }
+  argo-controller:               { charm: argo-controller,         channel: latest/edge, scale: 1 }
+  argo-server:                   { charm: argo-server,             channel: latest/edge, scale: 1 }
+  dex-auth:                      { charm: dex-auth,                channel: 2.28/edge, scale: 1, trust: true }
+  envoy:                         { charm: envoy,                   channel: latest/edge, scale: 1 }
+  istio-ingressgateway:          { charm: istio-gateway,           channel: 1.5/edge, scale: 1, trust: true}
```

Comment:
I don't think we're planning on having `istio-gateway` be compatible with the 1.5 track - `istio-gateway` is something introduced with 1.11

Reply:
In the end we decided to just publish the istio-ingressgateway into this track to make the whole thing consistent. It is still a istio gateway charm egress is just not enabled.